<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportsSettings</class>
 <widget class="QDialog" name="ReportsSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>353</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Настройки отчетов</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidgetReportTypes">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="TicketReports">
      <attribute name="title">
       <string>Квитанции</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="0">
        <widget class="QWidget" name="widgetJobListRep" native="true">
         <property name="minimumSize">
          <size>
           <width>586</width>
           <height>82</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QWidget" name="widgetCashReceiptRep" native="true">
         <property name="minimumSize">
          <size>
           <width>586</width>
           <height>83</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="widgetTicketRep" native="true">
         <property name="minimumSize">
          <size>
           <width>586</width>
           <height>83</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxBranch">
         <property name="title">
          <string>Текущий филиал:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="BranchSelectionWidget" name="branchSelectionWidget" native="true">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ProductsReports">
      <attribute name="title">
       <string>Товары</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QWidget" name="widgetPriceTagRep" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="widgetCashCheckRep" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButtonOk">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BranchSelectionWidget</class>
   <extends>QWidget</extends>
   <header>branchselectionwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
